File: Python/Program/Human.py
class Human:

    __nik = ""
    __nama = ""
    __gender = ""

    def __init__(self, nik="", nama="", gender=""):
        self.__nik = nik
        self.__nama = nama
        self.__gender = gender

    def set_nik(self, nik):
        self.__nik = nik

    def get_nik(self):
        return self.__nik

    def set_nama(self, nama):
        self.__nama = nama

    def get_nama(self):
        return self.__nama

    def set_gender(self, gender):
        self.__gender = gender

    def get_gender(self):
        return self.__gender

    def display_data(self):
        print("| NIK                 : ", self.__nik)
        print("| Nama                : ", self.__nama)
        print("| Jenis Kelamin       : ", self.__gender)

File: CPP/Program/Human.cpp
#include <bits/stdc++.h>
using namespace std;

class Human
{

private:
    string nik;
    string nama;
    string gender;

public:
    // Constructor
    Human()
    {
        this->nik = "";
        this->nama = "";
        this->gender = "";
    }

    Human(string nik, string nama, string gender)
    {
        this->nik = nik;
        this->nama = nama;
        this->gender = gender;
    }

    // Setter & Getter

    void setNik(string nik)
    {
        this->nik = nik;
    }

    string getNik()
    {
        return nik;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    string getNama()
    {
        return nama;
    }

    void setGender(string gender)
    {
        this->gender = gender;
    }

    string getGender()
    {
        return gender;
    }

    void displayDaya()
    {
        cout << "| NIK              : " << this->getNik() << endl;
        cout << "| Nama             : " << this->getNama() << endl;
        cout << "| Jenis Kelamin    : " << this->getGender() << endl;
    }

    // Destructor
    ~Human() {}
};